openapi.yaml

```diff
@@ -17,69 +17,116 @@ openapi: 3.0.3
 servers:
   - url: http://template/v1
 
 tags:
   - name: Configuration
-    description: Configuration stuff for app template
+    description: Configure access to Kentix devices
     externalDocs:
-      url: https://github.com/eliona-smart-building-assistant/app-template
+      url: https://github.com/eliona-smart-building-assistant/kentix-app
 
 paths:
-  /examples:
+  /configs:
     get:
       tags:
         - Configuration
-      summary: Get example configuration
-      description: Gets information about example.
-      operationId: getExamples
+      summary: Get all Kentix configurations
+      description: Gets information about all Kentix configurations
+      operationId: getConfigurations
       responses:
         200:
-          description: Successfully returned example configuration
+          description: Successfully returned configured Kentix configurations
           content:
             application/json:
               schema:
                 type: array
                 items:
-                  $ref: '#/components/schemas/Example'
+                  $ref: '#/components/schemas/Configuration'
     post:
       tags:
         - Configuration
-      summary: Creates an example configuration
-      description: Creates an example configuration.
-      operationId: postExample
+      summary: Creates a Kentix configuration
+      description: Creates a Kentix configuration.
+      operationId: postConfiguration
       requestBody:
         content:
           application/json:
             schema:
-              $ref: '#/components/schemas/Example'
+              $ref: '#/components/schemas/Configuration'
       responses:
         201:
-          description: Successfully created an example configuration
+          description: Successfully created a new Kentix configuration
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/Example'
+                $ref: '#/components/schemas/Configuration'
 
 components:
   schemas:
-
-    Example:
+    Configuration:
       type: object
-      description: Simply a example configuration
+      description: Each configuration defines access to one Kentix device.
       properties:
         id:
           type: integer
           format: int64
-          description: A id identifying the example configuration
+          description: Internal identifier for the configured device (created automatically). This identifier have to use always if you remove or update existing configured endpoints.
+          readOnly: true
+          nullable: true
+        address:
+          type: string
+          format: string
+          description: IP or hostname of the Kentix device
+          example: 10.10.10.101
+        apiKey:
+          type: string
+          description: Kentix API key
+        enable:
+          type: boolean
+          description: Flag to enable or disable the device
```

Comment:
With this flag you don't disable the device directly. You disable the data-fetching for this device. I think this comment can mislead users in Eliona.